main_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/gofiber/fiber/v2/utils"
 	"github.com/golang-jwt/jwt/v5"
 
-	jwtware "github.com/gofiber/jwt/v3"
+	jwtware "github.com/gofiber/jwt/v4"
```

Comment:
This should be /v5 now

Reply:
:slightly_smiling_face: check the import one more time.